tools-local/tasks/BuildTools.Publish/AzureBlobLease.cs

```diff
@@ -0,0 +1,183 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
```

Comment:
Should the namespaces be at least updated here?  They are rather confusing in this context.

Reply:
Hmm, I'm already adding some info to the commit desc that this is a verbatim copy, I'll add a README.md as well so that future readers don't need to think to look at the commit.

This namespace calls back to where the file exists in BuildTools. I'll see if splitting these files into a different dir makes sense, I think it would. Knowing which BuildTools project these came from might not actually end up useful, but I don't want to wipe out the info.